ansible/configs/rhel8lab/post_software.yml

```diff
@@ -35,6 +35,59 @@
           bookbag_os_username: "{{ guid }}-user"
           bookbag_os_password: "{{ hostvars.localhost.heat_user_password }}"
           bookbag_os_project_name: "{{ guid }}-project"
+          
+# for testing user info, from
+# https://github.com/redhat-cop/agnosticd/blob/d4e62c36ce7b55422aaca3adbb0db3b115faaab7/ansible/configs/ocp4-cluster/post_software.yml#L293-L315
+- name: Step 005.6 Print Student Info
+  hosts: localhost
+  gather_facts: false
+  become: false
+  tasks:
+  - name: Print Student Information
+    when: install_student_user | bool
+    block:
+    - name: Store bastion hostname as a fact
+      set_fact:
+        bastion_hostname: "{{groups['bastions'].0 }}"
+    - name: print out user.info
```

Comment:
Please use the `agnosticd_user_info` module for this. 

Refer to https://github.com/redhat-cop/agnosticd/blob/development/ansible/configs/ocp4-disconnected-osp-lab/post_software.yml#L10-L39 for example usage.